scripts/cybershake/add_to_mgmt_queue.py

```diff
@@ -33,6 +33,7 @@
         "--error",
         type=str,
         help="Errors that occurred during the execution of the script.",
+        default=None,
```

Comment:
Optional arguments are None by default?

Reply:
yeah, this shouldnt effect anything, it is just for debugging purposes. ( was having some issue when running on old stuffs)